GeoFS-Autoland/keyboard-mapping
# Changed the way keypresses are detected (this kinda works, i guess)

**THIS VERSION DOES NOT INCLUDE ANY UI CHANGES. IT IS NOT READY FOR A RELEASE.**

To test:
1. Open a GeoFS tab and paste the code from `dist\keyboard_mapping.js` in the console.
2. Import the addKeybind function: `const addKeybind = keyboard_mapping.require('addKeybind')`.
3. Add a keybind: `addKeybind(keybindName: string, callback: (event: KeyboardEvent): void, keybind: {altKey: boolean, ctrlKey: boolean, shiftKey: boolean, code: string}): void`, where `code` represents the string returned from `event.code`. You can check the code for any key [here](https://keycode.info/)
